chalicelib/checks/wrangler_checks.py

```diff
@@ -2683,6 +2695,8 @@ def sync_users_oh_start(connection, **kwargs):
             del a_user['lab_score']
             if a_user.get('OH_lab'):
                 del a_user['OH_lab']
+            if a_user.get('log'):
```

Comment:
It probably doesn't happen or you would have already seen this issue but what happens if a_user['log'] is None or a falsy value - it wont be deleted from a_user.  I think likely the better way to do this kind of delete when removing items that are not real properties so leaving them in will cause the post or patch to fail is to test if the key exists at all and if it does delete it:
eg. 
`if 'OH_lab' in a_user:
      del a_user['OH_lab']
`
That way you never have a problem if a key exists but the value doesn't.